src/components/dashboard/WhoContent.native.js

```diff
@@ -71,6 +89,23 @@ const WhoContent = ({ styles, setValue, error, text, value, next, state }) => {
     }
   })
 
+  useEffect(() => {
+    findRecentlyUsed()
+  }, [contacts, recentlyUsed])
+
+  const findRecentlyUsed = () => {
+    let matches = []
+    if (recentlyUsed.length > 0 && contacts.length > 0) {
+      recentlyUsed.forEach(contact => {
+        matches.push(
+          contacts.filter(item => item.phoneNumbers[0] && contact.data.phoneNumber === item.phoneNumbers[0].number)
```

Comment:
you never save the phone number to the contact.data.phoneNumber

Reply:
I did not understand, I'm using contact.data.phoneNumber to compare with contacts phone number (item.phoneNumbers[0].number) to get complete contact object.

Reply:
recentlyUsed are feed items (according to the logic you implemented)
so each "contact" is a feed item, and in no stage is any value saved to "contact.data.phoneNumber" why do you expect the feed item to have a phonenumber?

I would also rename "recentlyUsed" to "recentFeedItems", "contact" to "feedItem", "item" to "contact"

Reply:
contact.data.phoneNumber appears inside the object when a transaction that includes interaction between users completed. Before only include a user name, but I edited to also include the phone number. 
I will follow your suggestion, I think that rename
![Screen Shot 2020-03-18 at 10 11 45 AM](https://user-images.githubusercontent.com/59056508/76985857-9f8de280-691f-11ea-8b32-3844cbbaa2e9.png)
 is a good idea